src/main/java/org/musicbrainz/search/solrwriter/MBXMLWriter.java

```diff
@@ -381,13 +386,32 @@ public void write(Writer writer, SolrQueryRequest req, SolrQueryResponse res)
 			xmlList.add(unmarshalledObj);
 		}
 
+		XMLGregorianCalendar now = getNow();
+
+		Metadata metadata = metadatalistwrapper.getCompletedMetadata();
+		metadata.setCreated(now);
+
 		StringWriter sw = new StringWriter();
 		try {
-			marshaller.marshal(metadatalistwrapper.getCompletedMetadata(), sw);
+			marshaller.marshal(metadata, sw);
 		} catch (JAXBException e) {
 			e.printStackTrace();
-			return;
```

Comment:
Do we want to quietly drop errors?

Reply:
No we don't, and we also don't want to write some half-baked data, so I've added the return statement back.